client_admin/pulp/client/admin/tasks.py

```diff
@@ -60,6 +65,10 @@ def list(self, **kwargs):
         """
 
         self.context.prompt.render_title('Tasks')
+        if kwargs.get('all') and kwargs.get('state'):
```

Comment:
A nice pattern is to save the ``PulpCliFlag`` or ``PulpCliOption`` instance on your class, where you instantiate it, for example:

```
self.all_flag = PulpCliFlag(**stuff)
```

and then here (and in the tests) you can access the keyword directly like so:

```
if kwargs.get(self.all_flag.keyword) and kwargs.get(self.state_option.keyword):
```

This avoids having to track the keyword sting externally in some other constant, or by typing it in each time like you've done here.